initial-setup/initial-setup.spec

```diff
@@ -0,0 +1,356 @@
+%if 0%{?qubes_builder}
+%define _sourcedir %(pwd)/initial-setup
+%endif
+
+Summary: Initial system configuration utility
+Name: initial-setup
+URL: https://fedoraproject.org/wiki/InitialSetup
+Version: 0.3.37
+Release: 1%{?dist}
+Epoch:   1000
```

Comment:
Instead of importing the whole `initial-setup` package, could you create a small package with just `run-initial-setup` script and appropriate systemd service? Then enable only this one in anaconda (as currently done).